README.md

```diff
@@ -42,4 +42,8 @@ module.exports = (on, config) => {
 ```
 
 ## Options
-This plugin takes two paramaters. The first parameter (which is mandatory) is the Cypress config object and the other is an optional [dotenv](https://www.npmjs.com/package/dotenv#config) config object.
+This plugin takes three paramaters. The first parameter (which is mandatory) is the Cypress config object. 
+
+The second is an optional [dotenv](https://www.npmjs.com/package/dotenv#config) config object.
+
+The third is an optional [all] parameter, which is set to false by default. If set to true, it returns all available environmental variables, not limited to those prefixed with CYPRESS_.
```

Comment:
Would you mind adding in here that the 3rd parameter is a boolean, please?

Reply:
Updated.